torch/distributed/elastic/rendezvous/api.py

```diff
@@ -43,93 +42,80 @@ class RendezvousHandler(abc.ABC):
                  so the APIs may change!
     """
 
-    @abc.abstractmethod
+    @abstractmethod
     def get_backend(self) -> str:
-        """
-        Return the string representation of the rendezvous handler.
-        """
-        pass
+        """Returns the name of the rendezvous backend."""
 
-    @abc.abstractmethod
+    @abstractmethod
     def next_rendezvous(
         self,
     ) -> Tuple[Store, int, int]:
-        """
-        Main entry-point into the rendezvous barrier.
-        Blocks until the rendezvous is complete (and the current
-        process is included in the formed worker group), or a timeout occurs, or
+        """Main entry-point into the rendezvous barrier.
+
+        Blocks until the rendezvous is complete and the current process is
+        included in the formed worker group, or a timeout occurs, or the
         rendezvous was marked closed.
 
-        Returns: a tuple of (``c10d Store``, ``rank``, ``world size``)
+        Returns:
+            A tuple of C10d `Store`, `rank`, and `world size`.
 
         Raises:
-            RendezvousClosedError - if rendezvous for the current job is closed.
-            RendezvousTimeoutError - on timeout
+            RendezvousClosedError:
+                The rendezvous is closed.
+            RendezvousConnectionError:
+                The connection to the rendezvous backend has failed.
+            RendezvousStateError:
+                The rendezvous state is corrupt.
+            RendezvousTimeoutError:
+                The rendezvous did not complete on time.
         """
-        pass
 
-    @abc.abstractmethod
+    @abstractmethod
     def is_closed(self) -> bool:
+        """Checks whether the rendezvous has been closed.
+
+        A closed rendezvous means all future attempts to re-rendezvous within
+        same job will fail.
+
+        `is_closed()` and `set_closed()` have semantics of eventual propagation
+        and should not be used for synchronization. The intention is that if at
+        least one node decides the job is finished, it will close the rendezvous
+        and other nodes will soon observe this and stop running as well.
         """
-        Checks whether rendezvous for current job has been closed,
-        which means all future attempts to re-rendezvous (within same job) will
-        fail.
-
-        .. note:: ``is_closed`` and ``set_closed`` have semantics of eventual
```

Comment:
Is there a reason this note was removed?

Reply:
The note is still there actually. I just removed the "note::" marking and included the description in the main description.